packages/components/src/internal/components/editable/models.ts

```diff
@@ -212,118 +272,102 @@ export class EditorModel
+    getColumnValues(fieldKey: string): List<List<ValueDescriptor>> {
+        const colIdx = this.orderedColumns.findIndex(colName => colName === fieldKey);
         if (colIdx === -1) {
-            console.warn(`Unable to resolve column "${columnName}". Cannot retrieve column values.`);
+            console.warn(`Unable to resolve column with fieldKey "${fieldKey}". Cannot retrieve column values.`);
             return List();
         }
 
         const values = List<List<ValueDescriptor>>().asMutable();
         for (let i = 0; i < this.rowCount; i++) {
-            values.push(this.getValue(colIdx, i));
+            values.push(this.getValue(fieldKey, i));
         }
 
         return values.asImmutable();
     }
 
-    getRawDataFromModel(
-        queryModel: QueryModel,
-        displayValues?: boolean,
-        forUpdate?: boolean,
-        forExport?: boolean
-    ): List<Map<string, any>> {
-        return this.getRawDataFromGridData(
-            fromJS(queryModel.rows),
-            fromJS(queryModel.orderedRows),
-            queryModel.queryInfo,
-            displayValues,
-            forUpdate,
-            forExport
-        );
-    }
-
-    /** @deprecated Use getRawDataFromModel() instead. */
-    getRawDataFromGridData(
-        data: Map<any, Map<string, any>>,
-        dataKeys: List<any>,
-        queryInfo: QueryInfo,
-        displayValues = true,
-        forUpdate = false,
-        forExport?: boolean
-    ): List<Map<string, any>> {
-        let rawData = List<Map<string, any>>();
-        const columnMap = this.columns.reduce((map, fieldKey) => {
-            const col = queryInfo.getColumn(fieldKey);
-            // Log a warning but still retain the key in the map for column order
-            if (!col) console.warn(`Unable to resolve column "${fieldKey}". Cannot retrieve raw data.`);
-            return map.set(fieldKey, col);
-        }, OrderedMap<string, QueryColumn>());
-
-        for (let rn = 0; rn < dataKeys.size; rn++) {
-            let row = Map<string, any>();
-            columnMap.valueSeq().forEach((col, cn) => {
-                if (!col) return;
-                const values = this.getValue(cn, rn);
-
-                // Some column types have special handling of raw data, such as multi value columns like alias,
-                // so first check renderer for how to retrieve raw data
-                let renderer;
-                if (col.columnRenderer) {
-                    renderer = getQueryColumnRenderers()[col.columnRenderer.toLowerCase()];
-                }
+    /**
+     * Formats the values for an entire row into a Map<string, any>
+     * @param rowIdx
+     * @param displayValues
+     */
+    getRowValue(rowIdx: number, displayValues = true): Map<string, any> {
+        let row = Map<string, any>();
+
+        // TODO: NEED REVIEWER INPUT. Is it safe to use this.columnMap below? It includes pk columns (which we used to
```

Comment:
I'd say let's go with it and let the callers figure out if things need to be stripped out.